packages/expo-clipboard/src/web/WebUtils.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Converts base64-encoded data to a `Blob` object.
+ * @see https://stackoverflow.com/a/20151856
+ */
+export function base64toBlob(base64Data: string, contentType: string): Blob {
+  contentType = contentType || '';
+  const sliceSize = 1024;
+  const byteCharacters = atob(base64Data);
+  const bytesLength = byteCharacters.length;
+  const slicesCount = Math.ceil(bytesLength / sliceSize);
+  const byteArrays = new Array(slicesCount);
+
+  for (let sliceIndex = 0; sliceIndex < slicesCount; ++sliceIndex) {
+    const begin = sliceIndex * sliceSize;
+    const end = Math.min(begin + sliceSize, bytesLength);
+
+    const bytes = new Array(end - begin);
+    for (let offset = begin, i = 0; offset < end; ++i, ++offset) {
+      bytes[i] = byteCharacters[offset].charCodeAt(0);
+    }
+    byteArrays[sliceIndex] = new Uint8Array(bytes);
+  }
+  // I cannot use `@ts-expect-error` here because some environments consider this correct:
+  // expo-module build - OK,
+  // expo-module test - error
+  // @ts-ignore `Blob` from `lib.dom.d.ts` and the one from `@types/react-native` differ somehow
+  return new Blob(byteArrays, { type: contentType });
+}
+
+/**
+ * Converts blob to base64-encoded string with Data-URL prefix.
+ */
+export function blobToBase64Async(blob: Blob): Promise<string> {
+  return new Promise((resolve, _) => {
+    const reader = new FileReader();
+    reader.onloadend = () => resolve(reader.result as string);
+    reader.readAsDataURL(blob);
+  });
+}
+
+export function getImageSizeFromBlobAsync(blob: Blob): Promise<[width: number, height: number]> {
```

Comment:
I don't see a reason for using tuple here 🤔 
```suggestion
export function getImageSizeFromBlobAsync(blob: Blob): Promise<{ width: number, height: number }> {
```